src/index.html

```diff
@@ -15,8 +15,93 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="/src/images/logo.png"
+        alt="logo"
+      />
+      <nav class="nav">
+        <ul class="list">
+          <li class="list-item">
+            <a
+              href="index.html"
+              class="list-link"
```

Comment:
Fix classes everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="index.html"
              class="nav__link"
```

Reply:
Also add this class for the first link 
<img width="809" alt="image" src="https://github.com/user-attachments/assets/f6c44518-c391-4f5a-9d13-f333c56378fe">